functions/index.js

```diff
@@ -46,3 +46,27 @@ exports.incrementReflectionChoiceCounters = functions.firestore
       await choiceRef.update({ count: FieldValue.increment(1) });
     }
   });
+
+// Remove email field from users collection
+
+exports.removeEmailFieldFromUsersOnCreate = functions.firestore
+  .document('users/{docId}')
+  .onCreate(async (snap, context) => {
+    const { email } = snap.data();
+    if (!email) return;
+    const userRef = snap.ref;
+    await userRef.update({
+      email: FieldValue.delete(),
+    });
+  });
+
+exports.removeEmailFieldFromUsersOnUpdate = functions.firestore
```

Comment:
Propose to remove this second function, as it is not necessary. Email is only collected during record creation, and hence it is not possible for it to "appear" during record update.